svthalia/concrexit
# add pushnotifications for thabloid

Closes #3615 .



### Summary

Added the feature that a pushnotification is send when a new thabloid is uploaded. 
### How to test

1. Go to admin and upload a thabloid
2. See in the admin that under messages (for pushnotifications) a message for that thabloid has been added
